.agents/skills/webawesome/references/components/accordion-item.md

```
@@ -0,0 +1,116 @@
+To import this component for React 18 or below, use the following code:
+
+```js
+import WaAccordionItem from '@awesome.me/webawesome/dist/react/accordion-item/index.js';
+```
+
+### Slots
+
+| Name | Description |
+| --- | --- |
+| (default) | The accordion item's body content. |
+| \`icon\` | \`
```

markdownlint-cli2 (0.23.2)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)